emacs/caml.el

```diff
@@ -1137,57 +1097,7 @@ to the end.
 (defun caml-in-comment-p ()
   "Returns non-nil if point is inside a caml comment.
 Returns nil for the parenthesis opening a comment."
```

Comment:
What about this comment that the expected result on the opening parenthesis is `nil`, does the built-in Emacs function also respect that? (Otherwise I guess you could also check whether the position to the left is also within the comment.)

Reply:
So it works as the comment said it should, if I understand correctly.

Reply:
Yep, that's correct. After this change, we still return nil when point (the cursor) is on the opening `(`.